src/contexts/index.ts

```diff
@@ -0,0 +1,55 @@
+import { contextFactory } from 'config/context';
+import LocalStorageService from 'services/LocalStorageService';
+
+export interface State {
+  isAuth: boolean;
+  language: string;
+}
+
+export const INITIAL_STATE = {
+  isAuth: !!LocalStorageService.getValue('token') || false,
```

Comment:
It is preferable to get this information inside a useEffect when loading the app component